student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
     super(name: name, age: age, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.add_student(self) unless classroom.students.include?(self)
```

Comment:
- [ ] @Benawi Great work implementing the change requested however you did it wrongly, kindly consider adding a new method classroom that accomplishes the belong-to the association between the student and the classroom.
```suggestion
    def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
    end
```